app/src/main/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -202,10 +214,17 @@ class MessagingService : FirebaseMessagingService() {
         }
     }
 
-    private suspend fun getImageBitmap(url: String): Bitmap? = withContext(Dispatchers.IO) {
+    private suspend fun getImageBitmap(url: URL?, requiresAuth: Boolean = false): Bitmap? = withContext(Dispatchers.IO) {
+        if (url == null)
+            return@withContext null
```

Comment:
Kotlin is confusing.

Reply:
Definitely takes some getting used to, haha